proto/p4/config/p4info.proto

```diff
@@ -136,8 +138,22 @@ message ActionProfile {
 
 message Counter {
   Preamble preamble = 1;
-  // 0 (default value) if the counter is not direct, the id of the table to
-  // which the counter is attached otherwise
+  // in the current P4 spec the attribute is called 'type', I believe 'unit' is
+  // more appropriate for this and I hope to see the spec updated
+  enum Unit {
```

Comment:
Let's move `Unit` outside of this message (and rename it `CounterUnit`) so it can be shared by direct & indirect?